templates/impala/base.html

```diff
@@ -55,9 +58,15 @@
         {% if switch_is_active('fxa-auth') %}
           data-fxa-config="{{ fxa_config()|json }}"
         {% endif %}
+        {% if waffle.flag('restyle') %}
+          id="restyle"
```

Comment:
There's a risk here that we add might id in the bodyattrs block below. You might want to add a test for that or check for the presence of a data attr instead.